numba_dppy/examples/sum_reduction_recursive_ocl.py

```diff
@@ -79,13 +83,16 @@ def sum_recursive_reduction(size, group_size, Dinp, Dpartial_sums):
     return result
 
 
-def get_context():
-    if dpctl.has_gpu_queues():
-        return "opencl:gpu"
-    elif dpctl.has_cpu_queues():
-        return "opencl:cpu"
-    else:
-        raise RuntimeError("No device found")
+def get_device():
```

Comment:
We should move this function inside _helper.py

Reply:
done.